edx_adapt/data/course_repository.py

```diff
@@ -101,7 +101,7 @@ def post_interaction(self, course_id, problem_name, user_id, correct, attempt, u
 
         old_attempt = [x for x in self.get_raw_user_data(course_id, user_id)
                        if x['problem']['problem_name'] == problem_name and x['type'] == 'response'
-                          and x['attempt'] == attempt]
```

Comment:
The same as above.
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html